sdk/v1/client_test.go

```diff
@@ -44,9 +44,13 @@ func newMockHTTPClient(r *http.Response, err error) *dphttp.ClienterMock {
 	}
 }
 
-func newSearchReindexClient(httpClient *dphttp.ClienterMock) *Client {
+func newSearchReindexClient(t *testing.T, httpClient *dphttp.ClienterMock) *Client {
 	healthClient := healthcheck.NewClientWithClienter(serviceName, testHost, httpClient)
```

Comment:
(sugestion): can we run this is parallel ? t.Parallel ?

Reply:
No this is not a test, just a function used in the unit tests, the reason for passing *testing.T was to fail the test inside this function instead of handling it in the actual unit tests for every occasion (duplication of code)